openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -27,8 +85,7 @@ def standard_cookie_settings(request):
         expires = None
     else:
         max_age = request.session.get_expiry_age()
-        expires_time = time.time() + max_age
-        expires = cookie_date(expires_time)
+        expires = _cookie_expires_from_max_age(max_age)
```

Comment:
Nit: The name `_cookie_expires_from_max_age` reads awkwardly and its purpose is confusing.  Maybe `_cookie_expiration_based_on_max_age`?  Not sure.
An alternative is to move the `max_age` line into the helper, and name it `_cookie_expiration_to_match_session()`, or something like that?

Reply:
In my next PR, I do reuse the `_cookie_expires_from_max_age` helper and the `max_age` comes from a different source in that 2nd usage.  I renamed it to `_cookie_expiration_based_on_max_age`.